torch/csrc/jit/script/module.h

```diff
@@ -363,15 +333,27 @@ struct TORCH_API Module {
 
   IValue create_class(const c10::QualifiedName& name, Stack stack) const;
 
-  Slot get_slot(size_t slot) const {
-    TORCH_CHECK(
-        slot < module_object()->slots().size(), "not a valid slot offset");
-    return Slot(module_object(), slot);
-  }
-
   size_t num_slots() const {
     return module_object()->slots().size();
   }
+  c10::optional<EntityType> entity_type(const std::string& name) const {
```

Comment:
Should these two methods be private? EntityType shouldn't be a public concept I think

Reply:
Unfortunately, it's now used in `slot_iterator_impl`, so I had to keep it public. Do you mind if I take a look at how this can be cleaned-up in a follow-up?